SABT_MainUnit/dictionary.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * dictionary.h
+ *
+ * Created: 6/18/2015 8:45:21 AM
+ *  Author: Susan Zuo(tzuo)
+ */ 
+
+
+#ifndef DICTIONARY_H_
+#define DICTIONARY_H_
+
+typedef struct dictionary dictionary_t;
+/**
+ *  a struct to store words loaded from SD card

```

Comment:
This is still work in progress. Eventually, the teacher should be able to create txt files with specific name on SD card, and the words in the those files will be used as the dictionary for specific mode. Will let you know when it's done